lib/engines/content_object_store/app/controllers/content_object_store/content_block_editions_controller.rb

```diff
@@ -6,6 +6,22 @@ def index
   def new
     if params[:block_type].blank?
       @schemas = ContentObjectStore::SchemaService.valid_schemas
+    else
+      @schema = ContentObjectStore::SchemaService.schema_for_block_type(params[:block_type].underscore)
+      @content_block_edition = ContentObjectStore::ContentBlockEdition.new(block_type: @schema.block_type)
     end
   end
+
+  def create
+    @schema = ContentObjectStore::SchemaService.schema_for_block_type(params[:content_object_store_content_block_edition][:block_type])
```

Comment:
Should we be using strong params for this object creation too? Right now it's simple enough as the method only takes one param but in future this might not be the case and it could then be messed with.

Reply:
Good call. It'll be difficult to "permit" a limited subset of fields within `content_object_store_content_block_edition`, but I've tweaked it so we have a `root_params` hash which calls `params.require(:content_object_store_content_block_edition)`